internal/provision/devices.go

```diff
@@ -130,3 +91,109 @@ func LoadDevices(path string, dic *di.Container) errors.EdgeX {
 
 	return nil
 }
+
+func loadDevicesFromFile(path string, serviceName string, lc logger.LoggingClient, addDevicesReq []requests.AddDeviceRequest) errors.EdgeX {
```

Comment:
Better to return `[]requests.AddDeviceRequest`
```suggestion
func loadDevicesFromFile(path string, serviceName string, lc logger.LoggingClient ) ([]requests.AddDeviceRequest errors.EdgeX) {
```